lib/music/queue/addToQueue.ts

```diff
@@ -0,0 +1,11 @@
+import postSpotifyData from "@/lib/postSpotifyData";
+
+const addToQueue = async (uri: string) =>
+  postSpotifyData(`https://api.spotify.com/v1/me/player/queue`, {
```

Comment:
Is this the same as `fetchClient()` ?

Reply:
Yes, and the entire queue folder is unused. I just deleted it.